runtime/src/bank.rs

```diff
@@ -1145,6 +1148,7 @@ impl Bank {
             transaction_log_collector: new(),
             feature_set: new(),
             drop_callback: RwLock::new(OptionalDropCallback(None)),
+            freeze_started: AtomicBool::new(fields.hash != Hash::default()),
```

Comment:
@ryoqun I think this is a safe change to the ABI, and safe functionality-wise

Reply:
yeah, I think this is ok.